### PR TITLE
modified layout to be responsive

### PR DESCRIPTION
**Pull request by pe1te3son**

### CODE REVIEW

File: MarvinLee_Portfolio_FEND/css/main.css [hunk 1]
```diff
@@ -1,8 +1,7 @@
-body, html {
-  margin: 0;
-  padding: 0;
-  width: 100%;
-  height: 100%;
+body {
+  max-width: 950px;
```
Comment (pe1te3son) [inline MarvinLee_Portfolio_FEND/css/main.css:2]: We don't want the page to be larger than 950px. Feel free to change with to what you like

File: MarvinLee_Portfolio_FEND/css/main.css [hunk 2]
```diff
@@ -53,13 +52,12 @@ body, html {
...
+.main-content {
   display: flex;
   flex-wrap: wrap;
 }
 
-.mainImage {
-  order: -1;
+.main-image-container {
   width: 100%;
 }
```
Comment (pe1te3son) [inline MarvinLee_Portfolio_FEND/css/main.css:63]: Do not use camelCase when naming you html classes. Instead join words with -

File: MarvinLee_Portfolio_FEND/css/main.css [hunk 3]
```diff
@@ -102,29 +100,21 @@ figcaption {
...
-  margin: 0 auto;
+.featured_image img {
+  width: 100%;
 }
 
-.featured_images_right {
-  order: 1;
-  width: 33.3333%;
-  margin: auto 0 auto 0;
-}
```
Comment (pe1te3son) [inline MarvinLee_Portfolio_FEND/css/main.css:112]: All 3 above have the same style. Ordering them is unnecessary. I have set them to 32% and flexbox will fill the gab

File: MarvinLee_Portfolio_FEND/index.html [hunk 4]
```diff
@@ -21,32 +21,38 @@ <h1 class="name">Marvin Lee</h1>
...
+                  <p class="project-info">My first project</p>
+                </a>
               </figcaption>
         </figure>
-        <figure class="featured_images_center">
+        <figure class="featured_image">
             <img class="images_center" src="images/sunflower.png" alt="appify">
               <figcaption class="featured_fig">
-                <h2><a href="#">SUNFLOWER</a></h2>
-                <p class="featured_phr"><a href="#">https://github.com/udacity/Sunflower</a></p>
```
Comment (pe1te3son) [inline MarvinLee_Portfolio_FEND/index.html:42]: You do not need to include actual links. Instead wrap h2 and short description in <a> tags.

File: MarvinLee_Portfolio_FEND/css/main.css (hunk 1 above)
Comment (MarvinLee3) [reply MarvinLee_Portfolio_FEND/css/main.css:2]: Usaually the web site is not lager than 950px?

File: MarvinLee_Portfolio_FEND/css/main.css (hunk 2 above)
Comment (MarvinLee3) [reply MarvinLee_Portfolio_FEND/css/main.css:63]: I thought I have to use camelCase when naming the html classes. Thank you for letting me know this tip.

**Pull request merged**